Tests using token authentication in requests are broken

When running `make tests`, 9 tests are failing because of the token authentication in requests. Sample : 
```python
------------------------------------------------------------------------------------- Captured log call --------------------------------------------------------------------------------------
ERROR    /opt/mozdef/envs/python/bin/py.test:triage_bot.py:176 Request failed: 'str' object has no attribute 'decode'
Traceback (most recent call last):
  File "/opt/mozdef/envs/mozdef/mq/plugins/triage_bot.py", line 174, in update_alert_status
    resp = requests.post(url, json=payload, auth=jwt_auth)
  File "/opt/mozdef/envs/python/lib/python3.6/site-packages/requests/api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/opt/mozdef/envs/python/lib/python3.6/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/mozdef/envs/python/lib/python3.6/site-packages/requests/sessions.py", line 516, in request
    prep = self.prepare_request(req)
  File "/opt/mozdef/envs/python/lib/python3.6/site-packages/requests/sessions.py", line 459, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/opt/mozdef/envs/python/lib/python3.6/site-packages/requests/models.py", line 318, in prepare
    self.prepare_auth(auth, url)
  File "/opt/mozdef/envs/python/lib/python3.6/site-packages/requests/models.py", line 549, in prepare_auth
    r = auth(self)
  File "/opt/mozdef/envs/python/lib/python3.6/site-packages/requests_jwt.py", line 166, in __call__
    request.headers['Authorization'] = self._header_format % token.decode('ascii')
AttributeError: 'str' object has no attribute 'decode'
=========================================================================== 9 failed, 680 passed in 983.51 seconds ===========================================================================
make: *** [Makefile:64: run-tests] Error 1
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -49,3 +49,4 @@ grpc-google-iam-v1==0.12.3
 websocket-client==0.44.0
 maxminddb==1.5.2
 vine==1.3.0
+PyJWT==1.7.1

